Sources/Node/SchemaWrapper/SchemaWrapper.swift

```diff
@@ -0,0 +1,17 @@
+public protocol SchemaWrapper:
+    NodeConvertible,
+    PathIndexable,
+    Polymorphic,
+    Equatable,
+    ExpressibleByNilLiteral,
+    ExpressibleByBooleanLiteral,
+    ExpressibleByIntegerLiteral,
+    ExpressibleByFloatLiteral,
+    ExpressibleByStringLiteral,
+    ExpressibleByArrayLiteral,
+    ExpressibleByDictionaryLiteral
+{
+    var schema: Schema { get set }
```

Comment:
I think Schema was the closest I could come to having something that made sense to me, Value and Wrapped are fine internally, but since this value exists on its own I think it should have independent meaning. Something like Wrapped has context in an optional because it's wrapped by the optional, but having just `Wrapped` as an independent type I think isn't quite right.

Value might be able to work, but I think there's a better name for the underlying data that's more representative of what's going on. I'll brainstorm